src/Core/Program.cs

```diff
@@ -621,5 +645,88 @@ public static bool GetCommandLineFlagAsBool(string key, bool def)
             var mode => throw new InvalidDataException($"Command line flag '{key}' value of '{mode}' is not valid")
         };
     }
+    public static void PrintCommandLineHelp()
+    {
+        Console.WriteLine("""
+Usage: SwarmUI.dll [option]...
```

Comment:
running the built .dll directly is not the intended/proper way to run Swarm - `launch-linuxmac.sh` or `launch-windows.bat` should be used (both autoforward args)